framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -396,28 +405,32 @@ class BlockSynchronizationMechanism extends BaseSynchronizer {
 				currentRound,
 			);
 
-			const blockIds = (await this.storage.entities.Block.get(
-				{
-					height_in: heightList,
-				},
-				{
-					sort: 'height:asc',
-					limit: heightList.length,
-				},
-			)).map(block => block.id);
+			const blockIds = (
```

Comment:
is it because of formatting? can you fix it if its not a formatting issue?

Reply:
It's just prettier doing its things :D